database/drivers/postgres/parse.go

```diff
@@ -535,7 +537,8 @@ func queryColumnComments(log *log.Logger, db *sql.DB, schemaNames []string) ([]c
 		}
 
 		if c.Valid {
-			r.Comment = c.String
+			replaced := strings.Replace(c.String, "\n", " ", -1)
```

Comment:
What's the rationale for this change? Might a newline not be a significant part of the comment?

Reply:
If there is \n in comment in pg, gnorm will throw error: 

error calling makeTable: failed to render from pipe delimited bytes: record on line 4: wrong number of fields

This is about to fix it.

Reply:
I just ran into this issue with some of the comments in my database. Can this be merged?

Reply:
Then again, this might not be the right fix. I believe I ran into this issue when running `gnorm preview` on a comment containing `|` and `)`. Is there an existing issue for this?

Reply:
This pr is about to fix `\n` in comment. I didn't meet the `|` or `)` issue before.

Reply:
For `|` and `)`, may be needs more `strings.Replace`